tests/devices/unit_tests/test_zocalo_results.py

```diff
@@ -204,14 +206,20 @@ async def test_zocalo_results_trigger_log_message(
     mock_wrap_subscribe, mock_logger, RE: RunEngine
 ):
     zocalo_results = ZocaloResults(
-        name="zocalo", zocalo_environment="dev_artemis", timeout_s=2
+        name="zocalo",
+        zocalo_environment="dev_artemis",
+        timeout_s=2,
```

Comment:
Could: It's not great that we're waiting on this timeout for the test. Ideally we would mock it out so there's no wait on it but if not could we at least drop it down to as low as possible?